app/views/items/_index.html.erb

```diff
@@ -0,0 +1,18 @@
+<table>
+  <tr>
+    <th>Name</th>
+    <th>Description</th>
+    <th>created</th>
+    <th></th>
+  </tr>
+
+  <%- @items.order(created_at: :desc).each do |item| -%>
```

Comment:
Isn't @items already ordered from the AR query on the controller? I believe this is redundant. These queries in general should happen on the controller so that you can pass just `@items` to the view.